Description: fix POD errors
 remove extraneous =cut
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=85192
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=85192
Author: Jonathan Yu <jawnsy@cpan.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-05-11

--- a/lib/Gearman/Client.pm
+++ b/lib/Gearman/Client.pm
@@ -419,5 +419,3 @@
  Jonathan Steinert (hachi@cpan.org)
 
 =cut
-
-=cut
